Description: Fix incorrect count of netwm atoms
 The commit added 4 new atoms but only incremented the atom count by 3.
 Which can cause a crash as soon as the NET classes are used.
Author: Martin Gräßlin
Origin: upstream, https://git.reviewboard.kde.org/r/120539
Bug: https://bugs.kde.org/show_bug.cgi?id=339771
Last-Update: 2014-10-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/src/netwm.cpp b/src/netwm.cpp
index 3107a2388c2fb3f1a8ad97e466be40e6977e064b..7b398d46e9da2c85a96ab67b6c0f97b22ceae4a3 100644
--- a/src/netwm.cpp
+++ b/src/netwm.cpp
@@ -386,7 +386,7 @@ static QByteArray get_atom_name(xcb_connection_t *c, xcb_atom_t atom)
 }
 #endif
 
-static const int netAtomCount = 91;
+static const int netAtomCount = 92;
 
 static void create_netwm_atoms(xcb_connection_t *c)
 {
@@ -498,6 +498,7 @@ static void create_netwm_atoms(xcb_connection_t *c)
     };
 
     assert(!netwm_atoms_created);
+    assert(sizeof(atoms)/sizeof(atoms[0]) == netAtomCount + 1);
 
     // Send the intern atom requests
     xcb_intern_atom_cookie_t cookies[netAtomCount];
